chronos.nimble

```diff
@@ -32,10 +32,11 @@ proc run(args, path: string) =
 
 task test, "Run all tests":
   for args in [
-      "-d:useSysAssert -d:useGcAssert",
-      "-d:chronosStackTrace -d:chronosStrictException",
+      "-d:debug -d:chronosDebug",
+      "-d:debug -d:chronosPreviewV4",
```

Comment:
-d:debug -d:useSysAssert -d:useGcAssert is also important, because i'm not sure if its available for `release` mode.

Reply:
bbd706446e4ed19c39353f728ef989b1e5bc5422

Reply:
bbd706446e4ed19c39353f728ef989b1e5bc5422